pkgs/development/python-modules/kubernetes-validate/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pyyaml,
+  jsonschema,
+  typing-extensions,
+  importlib-resources,
+  packaging,
+  referencing,
+  flake8,
+  pytest,
+  pep8-naming,
+  tox,
+  wheel,
+}:
+buildPythonPackage rec {
+  pname = "kubernetes-validate";
+  version = "1.31.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "willthames";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-vxsftuipw0rHQIngxKlPHwBIW+rYAjfnEEaJDKmPyfQ=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    pyyaml
+    jsonschema
+    typing-extensions
+    importlib-resources
+    packaging
+    referencing
+  ];
+
+  nativeCheckInputs = [
+    flake8
+    pytest
```

```suggestion
    pytestCheckHook
```

We don't care about liniting thus those modules could be removed.